JS/Lesson22/goksel/style.css

```diff
@@ -0,0 +1,105 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    font-family: sans-serif;
```

Comment:
Use :root or body element to set font-family for the project. Using * selector would set font-family **individually** for every element with is not very efficient approach.